roles/senior_engineer.md

```diff
@@ -21,9 +21,9 @@ Further required experience:
 - Pair programming as we pair around 50% of the time
 - Agile practices such as Scrum, XP, and/or Kanban
 - Working with a range of SQL and NoSQL databases
-- Infrastructure as code tools such as Ansible, Chef, Puppet, and/or Saltstack
+- Infrastructure as code tools such as Ansible, Chef, Puppet, Terraform and/or Saltstack
 - Debugging a range of applications
-- Experience debugging infrastructure (AWS/Heroku/Linux/HTTP)
+- Experience debugging infrastructure (AWS/Heroku/Linux/HTTP, GCP and Azure)
```

Comment:
Should probably be consistent formatting:

AWS/Heroku/Linux/HTTP/GCP/Azure or
AWS, Heroku, Linux, HTTP, GCP, and Azure